renderer/src/features/registry-servers/components/form-run-from-registry/allowed-protocols-field.tsx

```diff
@@ -0,0 +1,54 @@
+import { Label } from '@/common/components/ui/label'
+
+interface AllowedProtocolsFieldProps {
+  field: {
+    value?: string[]
+    onChange: (value: string[]) => void
+  }
+}
+
+export function AllowedProtocolsField({ field }: AllowedProtocolsFieldProps) {
+  return (
+    <div className="mt-6">
+      <Label htmlFor="allowed-protocols-group">Allowed Protocols</Label>
+      <div
+        id="allowed-protocols-group"
+        role="group"
+        aria-label="Allowed Protocols"
+      >
+        <div className="mt-2 flex items-center gap-4">
+          <label>
+            <input
+              type="checkbox"
```

Comment:
you can import and use shadcn checkbox https://ui.shadcn.com/docs/components/checkbox

Reply:
fixed